data.sql

```diff
@@ -10,4 +10,27 @@ INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_
 VALUES (3, 'Pikachu', DATE '2021-01-07', 1, false, 15.04);
 
 INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
-VALUES (4, 'Devimon', DATE '2017-05-12', 5, true, 11.0);
\ No newline at end of file
+VALUES (4, 'Devimon', DATE '2017-05-12', 5, true, 11.0);
+
+-- insert new data into the table 
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES (5, 'Charmander', DATE '2020-02-08', 0, false, 11.0);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES (6, 'Plantmon', DATE '2021-11-15', 2, true, 5.7);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES (7, 'Squirtle', DATE '1993-04-02', 3, false, 12.13);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES (8, 'Angemon', DATE '2005-06-12', 1, true, 45);
+
```

Comment:
- [ ] Well done on this section, however, these `weight_kg values` are meant to be negative, see? please make an update on it.

Reply:
Fixed.